### DISCUSSION TITLE
Aborted (core dumped)

### DISCUSSION THREAD

**Discussion by FRNK-BK** [Akaflieg-Freiburg/enroute#336] [General]
Hi Akaflieg, 

I compiled the linux desktop version from source and all went fine.
But program terminates immediately (does not start).

`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.`
`QCoreApplication::applicationFilePath: Please instantiate the QApplication object first`
`QSocketNotifier: Can only be used with threads started with QThread`
`KDSingleApplication requires a non-empty application name`
`Aborted (core dumped)`

I think the first lines are just warnings. But then: aborted.
Any hint what's the issue?
Should I post any logs/traces?

**Comment by kebekus** [comment]
@FRNK-BK Thank you for this message. I apologize for my late answer. I have been in China for business, with limited internet access lately. GitHub is totally out of reach in this country.

The issue you are pointing to is likely due to a misconfiguration in the Qt libraries; I have encountered similar problems when experimenting with flatpak. Please try to see if setting QT_QPA_PLATFORM=xcb does the trick. I am looking forward to your feedback.

Best wishes,

Stefan.
